Backend test failing on develop

Backend tests for `story_services_test` is failing on develop with the error:

```
FAIL: test_get_node_index_by_story_id_and_node_id (core.domain.story_services_test.StoryProgressUnitTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ankita/Desktop/opensrc/oppia/core/domain/story_services_test.py", line 665, in test_get_node_index_by_story_id_and_node_id
    self.STORY_1_ID, 'node_4')
AssertionError: Exception not raised

---------------------------------------------------------------------
FAILED    core.domain.story_services_test: 0 errors, 1 failures

Ran 56 tests in 1 test class.
(0 ERRORS, 1 FAILURES)
Traceback (most recent call last):
  File "scripts/backend_tests.py", line 373, in <module>
    main()
  File "scripts/backend_tests.py", line 369, in main
    '%s errors, %s failures' % (total_errors, total_failures))
Exception: 0 errors, 1 failures

```

This change was introduced in #6973. /cc @sophiewu6

--- a/core/domain/story_services_test.py
+++ b/core/domain/story_services_test.py
@@ -660,9 +660,9 @@ class StoryProgressUnitTests(StoryServicesUnitTests):
         # Tests error should be raised if story or node doesn't exist.
         with self.assertRaisesRegexp(
             Exception,
-            'Story node with id node_4 does not exist in this story.'):
+            'Story node with id node_5 does not exist in this story.'):
             story_services.get_node_index_by_story_id_and_node_id(
-                self.STORY_1_ID, 'node_4')
+                self.STORY_1_ID, 'node_5')
 
         with self.assertRaisesRegexp(
             Exception, 'Story with id story_id_2 does not exist.'):

